florestad/src/florestad.rs

```diff
@@ -638,3 +699,13 @@ impl From<Config> for Florestad {
         }
     }
 }
+
+fn create_tls_config(cert_path: &str, key_path: &str) -> io::Result<Arc<ServerConfig>> {
```

Comment:
nit: why is this function not inside `Florestad`. It's a bit weird having it as the only free floating method in this module.